packages/app/src/systems/Account/pages/Logout/Logout.tsx

```diff
@@ -42,12 +74,33 @@ export const Logout = () => {
           <Text color="gray11" css={{ mb: '$2' }}></Text>
         </Card>
       </Dialog.Description>
+      <ControlledField
+        control={control}
+        name="logoutConfirmation"
+        label={
+          <p>
+            Write <i>{logOutConfirmationPhrase}</i> to logout
+          </p>
+        }
+        render={({ field }) => (
+          <Input>
+            <Input.Field
+              {...field}
+              aria-label="Confirm Log Out"
+              placeholder={logOutConfirmationPhrase}
+            />
+          </Input>
+        )}
+      />
       <Dialog.Footer>
         <Button
           aria-label="Logout"
           onPress={handlers.logout}
           isLoading={isLoading}
-          isDisabled={isLoading}
+          isDisabled={
+            isLoading ||
+            form.getValues().logoutConfirmation !== logOutConfirmationPhrase
+          }
```

Comment:
can we move this into a const `isLogoutButtonDisabled` ?
```suggestion
          isDisabled={isLogoutButtonDisabled}
```